types/utils.go

```diff
@@ -29,3 +31,34 @@ func CheckForDuplicatesAndEmptyStrings(input []string) error {
 
 	return nil
 }
+
+// GetActivationHeight returns the minimum block height from which the BTC
+// staking protocol starts updating the voting power table.
+func GetActivationHeight(btcNetwork string) uint64 {
+	switch btcNetwork {
+	// The activation height might differ accordingly
+	// with the test that will be done and the block time
+	// that the validator sets in config.
+	case chaincfg.MainNetParams.Name:
+		// For mainnet considering we want 48 hours
+		// at a block time of 10s that would be 17280 blocks
+		// considering the upgrade for Phase-2 will happen at block
+		// 220, the mainnet activation height for btcstaking should
+		// be 17280 + 220.
+		return 17500
+	case chaincfg.SigNetParams.Name:
```

Comment:
In testnet, we would use Signet for btc but to simulate the exact behavior as the mainnet. What activation height do we want for that?

Reply:
To simulate the exact behavior would need to be the same height as mainnet 17500